.changeset/lucky-tigers-carry.md

```diff
@@ -0,0 +1,6 @@
+---
+'@aws-amplify/backend-data': minor
+'@aws-amplify/sandbox': minor
```

Comment:
why is sandbox included in changeset?

Reply:
The sandbox mode behaves differently when the GSI updates are detected. Once this PR takes effect, the table will be replaced instead of change the GSIs to achieve a faster deployment experience.

Reply:
That is fine, but the behavior change comes from `backend-data`, not sandbox.
In theory, it's possible to use new `sandobox` with old `backend-data` - in that case the old GSI mode will take effect.

We should remove sandbox from changeset.